docs/changelog/0024-upgrade-to-python-3.md

```diff
@@ -1,8 +1,8 @@
 <!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
 <!--See https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md for instructions-->
-# 23. Upgrade to Python 3
+# 24. Upgrade to Python 3
```

Comment:
Moving these files will break the link in the forum https://forum.dimagi.com/t/python-2-deprecation-notice-30-days-to-upgrade/6699

Is there a reason to not just fix the dates to when these were merged to master (I'm assuming they were in order)